src/components/calendar/calendar.jsx

```diff
@@ -0,0 +1,199 @@
+import React, {
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import arrowLeft from '../../svgs/arrow-left.svg';
+import arrowRight from '../../svgs/arrow-right.svg';
+import caretDown from '../../svgs/caret-down.svg';
+import IconButton from '../icon-button/icon-button.jsx';
+import Icon from '../icon/icon.jsx';
+import styles from './calendar.css';
+
+function getDateIterator(year, month) {
+  const firstDate = new Date(year, month, 1);
+  return new Date(year, month, 1 - firstDate.getDay());
+}
+
+function getHeadCell(iterator) {
+  const date = iterator.getDate();
+  const weekday = iterator.toLocaleDateString(undefined, {
+    weekday: 'narrow',
+  });
+
+  // This needs to occur at the end of this function
+  iterator.setDate(date + 1);
+
+  return (
+    <th className={styles.weekday}>
+      {weekday}
+    </th>
+  );
+}
+
+function getCellClassName(iterator, month, highlightedDate) {
+  if (
+    highlightedDate.getFullYear() === iterator.getFullYear()
+    && highlightedDate.getMonth() === iterator.getMonth()
+    && highlightedDate.getDate() === iterator.getDate()
+  ) return styles['highlighted-date'];
+
+  if (iterator.getMonth() === month) return styles.date;
+
+  return styles['not-current-date'];
+}
+
+function getCell(iterator, month, highlightedDate) {
+  const date = iterator.getDate();
+  const className = getCellClassName(iterator, month, highlightedDate);
+
+  // This needs to occur at the end of this function
+  iterator.setDate(date + 1);
+
+  return (
+    <td className={className}>
+      {date}
+    </td>
+  );
+}
+
+function Calendar(props) {
+  const highlightedDate = new Date(props.value ? `${props.value}T00:00` : Date.now());
+  const [year, setYear] = useState(highlightedDate.getFullYear());
+  const [month, setMonth] = useState(highlightedDate.getMonth());
+  // const [date] = useState(highlightedDate.getDate());
```

Comment:
Could we remove this line?

Reply:
@bennapp did you end up using this variable for the active index management? (or did we decide to explore using it?)

Reply:
Either way, @bennapp I removed this. If there is a merge conflict, feel free to bring it back.